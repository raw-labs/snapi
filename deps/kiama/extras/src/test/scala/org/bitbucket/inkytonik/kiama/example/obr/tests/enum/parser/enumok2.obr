(* Enumeration with two constants *)

PROGRAM EnumOk2 (i : INTEGER) : INTEGER;

VAR
   Binary : (One, TheOther);

BEGIN
END EnumOk2.
