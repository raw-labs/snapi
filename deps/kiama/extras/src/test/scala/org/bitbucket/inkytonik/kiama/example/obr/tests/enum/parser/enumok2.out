ObrInt(EnumOk2,Vector(IntParam(IdnDef(i)), EnumVar(IdnDef(Binary),Vector(EnumConst(IdnDef(One)), EnumConst(IdnDef(TheOther))))),Vector(),EnumOk2)
